### PR TITLE
Rent index and creation

### PR DESCRIPTION
**Pull request by HarryPerez**
## Screenshots
Captura de Pantalla 2019-06-03 a la(s) 17 27 06
Captura de Pantalla 2019-06-03 a la(s) 17 27 13
Captura de Pantalla 2019-06-03 a la(s) 17 27 23

## Trello Card
https://trello.com/c/C4VJi17k/28-crear-endpoint-de-creacion-de-rent-e-index

### CODE REVIEW

File: app/controllers/v1/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module V1
+  class RentsController < ApiController
+    include Wor::Paginate
+
+    def index
+      render_paginated Rent
+    end
+
+    def create
+      rent = Rent.create!(rent_params)
```
Comment (Gmarn) [inline app/controllers/v1/rents_controller.rb]: Hummmm, no! Este punto de exclamacion? :so-loco-vos: ?

Comment (Gmarn) [reply app/controllers/v1/rents_controller.rb]: Podemos itentar un flujo con new and save y manejar el caso de un reocrd invalid con `rescue_from`?

File: app/controllers/v1/rents_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module V1
+  class RentsController < ApiController
+    include Wor::Paginate
+
+    def index
+      render_paginated Rent
```
Comment (Gmarn) [inline app/controllers/v1/rents_controller.rb]: Clap 👍

File: app/controllers/v1/rents_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+module V1
+  class RentsController < ApiController
```
Comment (Gmarn) [inline app/controllers/v1/rents_controller.rb]: Question: Estos end points son authenticado?

**Review by Gmarn — CHANGES_REQUESTED** [review]

File: app/controllers/v1/rents_controller.rb (hunk 3 above)
Comment (HarryPerez) [reply app/controllers/v1/rents_controller.rb]: Yes why not?

File: app/controllers/v1/rents_controller.rb (hunk 2 above)
Comment (HarryPerez) [reply app/controllers/v1/rents_controller.rb]: 👏

File: app/controllers/v1/rents_controller.rb (hunk 1 above)
Comment (HarryPerez) [reply app/controllers/v1/rents_controller.rb]: tenor-229695651

Comment (HarryPerez) [reply app/controllers/v1/rents_controller.rb]: What did you want to say with `new` and `save`?

**Review by Gmarn — DISMISSED** [review]

**Review by Gmarn — APPROVED** [review]

**Pull request merged**